tests/rolling_test.go

```diff
@@ -379,6 +379,68 @@ var _ = Describe("Test Rolling", func() {
 			},
 		},
 		),
+		Entry("filter nodes by --version flag, ==full_version_string_1234", TestCase{
+			nodeConfiguration: [][]uint32{
+				{1, 2, 3},
+			},
+			nodeInfoMap: map[uint32]mock.TestNodeInfo{
+				1: {
+					Version: "full_version_string_1234",
+				},
+				2: {
+					Version: "full_version_string_1235",
+				},
+				3: {
+					Version: "full_version_string_1236",
+				},
+			},
+			steps: []StepData{
+				{
+					ydbopsInvocation: []string{
+						"--endpoint", "grpcs://localhost:2135",
+						"--verbose",
+						"--availability-mode", "strong",
+						"--user", mock.TestUser,
+						"--cms-query-interval", "1",
+						"run",
+						"--storage",
+						"--version", "==full_version_string_1234",
```

Comment:
Looks good to add some "table" tests for that.

For example, func TestParseSomething(t *testing.T) {}